### ISSUE TITLE
Incorrect conjugations of 'harbor'

### ISSUE THREAD

**Issue by Perspx** [SekouDiaoNlp/mlconjug3#47]
The conjugations of the English verb 'harbor' are incorrect with the 'r' dropped in all forms and an 'e' added to some erroneously.

Oddly enough, the conjugations for the British English spelling, _harbour_, are correct, and the conjugations for the US-English _color_ are correct. 

    import mlconjug3
    default_conjugator = mlconjug3.Conjugator("en")
    print(default_conjugator.conjugate("harbor").conjug_info)

Returns:

    OrderedDict([
        ('indicative', OrderedDict([
            ('indicative present', OrderedDict([
                ('1s', 'harboe'),
                ('2s', 'harboe'),
                ('3s', 'harboes'),
                ('1p', 'harboe'),
                ('2p', 'harboe'),
                ('3p', 'harboe')])),
            ('indicative past tense', OrderedDict([
                ('1s', 'harboed'),
                ('2s', 'harboed'),
                ('3s', 'harboed'),
                ('1p', 'harboed'),
                ('2p', 'harboed'),
                ('3p', 'harboed')])),
            ('indicative present continuous', OrderedDict([
                ('1s 1s', 'harboing'),
                ('2s 2s', 'harboing'),
                ('3s 3s', 'harboing'),
                ('1p 1p', 'harboing'),
                ('2p 2p', 'harboing'),
                ('3p 3p', 'harboing')])),
            ('indicative present perfect', OrderedDict([
                ('1s', 'harboed'),
                ('2s', 'harboed'),
                ('3s', 'harboed'),
                ('1p', 'harboed'),
                ('2p', 'harboed'),
                ('3p', 'harboed')]))])),
        ('infinitive', OrderedDict([
            ('infinitive present', 'to harboe')])),
            ('imperative', OrderedDict([
                ('imperative present', OrderedDict([
                    ('2s', 'harboe'),
                    ('1p', 'harboe'),
                    ('2p', 'harboe')]))]))])

(formatted for easy reading).

Tested on mlconjug 3.7.5

**Comment by SekouDiaoNlp** [comment]
Hi @Perspx .

Thanks for submitting the bug request, I will investigate the issue and release a new version with the fix as soon as possible.

**Comment by SekouDiaoNlp** [comment]
Hi @Perspx I just pushed the fix for this issue on the master branch here on GitHub.

While I was investigating the other issue with Spanish verbs, I discovered a new [issue](https://github.com/SekouDiaoNlp/mlconjug3/issues/50) where some Spanish verbs were missing some conjugated forms.

I will publish a fix for this new issue today or in the next couple of days and then I will release a new version of mlconjug3 on PyPi.

Thanks again for using mlconjug3 and for submitting bug reports.

Cheers,

SekouDiaoNlp.
